api/types/authority.go

```diff
@@ -557,6 +558,24 @@ type CertRoles struct {
 	Roles []string `json:"roles"`
 }
 
+// MarshalCertRoles marshal roles list to OpenSSH
+func MarshalCertRoles(roles []string) (string, error) {
```

Comment:
Do need MarshalCertRoles and UnmarshalCertRoles to be under api/?

Reply:
No.  I needed to move them to avoid cyclic dependencies with `lib/services`, but after some consideration I've moved this logic so it now lives alongside the cert encoding logic in `lib/sshca`.

Reply:
I thought you would say that. +1 for it not being under api/.